sdk/search/Azure.Search.Documents/samples/Sample05_IndexingDocuments.md

```diff
@@ -0,0 +1,141 @@
+# Azure.Search.Documents Samples - Indexing Documents
+
+If we want to build a search experience for a product catalog, we need to get
+all of that data into Azure Cognitive Search.  If your data lives in Azure
+Cosmos DB, Azure SQL Database, or Azure Blob Storage you can setup an indexer
+to do that for you.  But if your data lives elsewhere, this sample will show you
+how to move it into a search index.
+
+Let's start with a simple model type:
+
+```C# Snippet:Azure_Search_Documents_Tests_Samples_Sample05_IndexingDocuments_LegacyProduct
+public class Product
+{
+    [SearchableField(IsKey = true)]
```

Comment:
Nit: less expensive - though this is only a sample - to use a `SimpleField` here. It's not like this unique ID is useful for lexical searches.

Reply:
Fixed